nifi-toolkit/nifi-toolkit-assembly/pom.xml

```diff
@@ -190,7 +190,7 @@ language governing permissions and limitations under the License. -->
                     <plugin>
                         <groupId>com.spotify</groupId>
                         <artifactId>dockerfile-maven-plugin</artifactId>
-                        <version>1.3.5</version>
+                        <version>1.4.13</version>
```

Comment:
Is it worth defining this version in a higher level `pom.xml` and using it for all Docker builds?

Could consider the same with the base image details too so we keep them updated together in future, although maybe there's sufficient reason to keep them separate still?

Reply:
I agree that this is a good candidate for promotion to the root Maven configuration. It should be possible to define a plugin dependency with the version, which will avoid the need to set the version in all of these instances.